bindings/nodejs/lib/types/client/bridge/index.ts

```diff
@@ -22,11 +26,17 @@ import type {
     __GetProtocolParametersMethod__,
     __GetHealthMethod__,
     __GetNodeInfoMethod__,
-    __GetPeersMethod__,
     __PostBlockRawMethod__,
     __GetBlockRawMethod__,
     __GetIncludedBlockMethod__,
     __GetIncludedBlockMetadataMethod__,
+    __GetTransactionMetadataMethod__,
+    __GetCommitmentMethod__,
+    __GetUtxoChangesMethod__,
+    __GetUtxoChangesFullMethod__,
+    __GetCommitmentByIndexMethod__,
+    __GetUtxoChangesByIndexMethod__,
+    __GetUtxoChangesFullByIndexMethod__,
```

Comment:
Is there any order here that we adhere to?

Reply:
No